x-pack/plugins/lens/public/trigger_actions/open_lens_config/create_action_helpers.ts

```diff
@@ -52,13 +52,41 @@ export async function executeCreateAction({
   const defaultDataView = await deps.dataViews.getDefaultDataView({
     displayErrors: false,
   });
-  if (!isCompatibleAction || !defaultDataView) {
+
+  const getFallbackDataView = async () => {
+    const getIndicesList = async () => {
+      const indices = await deps.dataViews.getIndices({
+        showAllIndices: false,
+        pattern: '*',
+        isRollupIndex: () => false,
+      });
+      return indices.filter((index) => !index.name.startsWith('.')).map((index) => index.name);
+    };
+
+    const indices = await getIndicesList();
+    let indexName = indices[0];
+    if (indices.length > 0) {
+      if (indices.find((index) => index.startsWith('logs'))) {
+        indexName = 'logs*';
+      }
+    }
+
+    if (!indexName) return null;
+
+    const dataView = await deps.dataViews.create({ title: indexName });
+
+    return dataView;
+  };
+
+  const dataView = defaultDataView ?? (await getFallbackDataView());
```

Comment:
This allows to add new ES|QL panels to a dashboard when there is no data views

Reply:
Nice!